docs/source/examples/Layout Example.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipyleaflet import Map, basemaps, basemap_to_tiles, Heatmap, TileLayer\n",
+    "from ipywidgets import AppLayout\n",
```

Comment:
(I can't comment on the other notebook on github) in the other notebook, it could be relevant to add a note / comment in examples that require other widget libraries.

Reply:
Done!